modmesh/app/euler1d.py

```diff
@@ -46,6 +49,9 @@ def load_app():
 # Use the functions for more examples:
 ctrl.start()  # Start the movie
 ctrl.step()  # Stepping the solution
+
+# Note: you can enable profiling info by "profiling" option
+ctrl = mm.app.euler1d.run(interval=10, max_steps=50, profiling=True)
```

Comment:
The one-liner aforementioned https://github.com/solvcon/modmesh/pull/190#issuecomment-1383108362 .

Reply:
This is great!